node/pkg/allocateip/allocateip.go

```diff
@@ -479,11 +513,14 @@ func updateNodeWithAddress(ctx context.Context, c client.Interface, nodename str
 		}
 
 		_, err = c.Nodes().Update(ctx, node, options.SetOptions{})
-		if _, ok := err.(cerrors.ErrorResourceUpdateConflict); ok {
-			// Wait for a second and try again if there was a conflict during the resource update.
-			log.WithField("node", node.Name).WithError(err).Info("Error updating node, retrying.")
-			time.Sleep(1 * time.Second)
-			continue
+		if err != nil {
+			log.WithField("node", node.Name).WithError(err).Info("Error updating node")
```

Comment:
I think I'd put this log below the ResourceUpdateConflict branch and upgrade it to a warning - this indicates a real error trying to update the node, and putting it after the branch means we only get one of the two logs rather than both in the case that we hit a CAS error.

Reply:
Good point, will do! Thanks!